recorder.go

```diff
@@ -110,14 +111,33 @@ type Options struct {
 
 	// Set Verbose to true to enable more text logging.
 	Verbose bool
+
+	UseGRPC bool
```

Comment:
Ug, I hope we're not keeping this flag very long.
ᐧ

On Wed, Sep 7, 2016 at 2:45 PM, spoons <notifications@github.com> wrote:

> In recorder.go
> <https://github.com/lightstep/lightstep-tracer-go/pull/34#discussion_r77910065>
> :
>
> > @@ -110,14 +111,33 @@ type Options struct {
> >
> >  	// Set Verbose to true to enable more text logging.
> >  	Verbose bool
> > +
> > +	UseGRPC bool
>
> Perhaps this for prettiness
>
> UseGRPC bool `yaml:"use_grpc"`
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/lightstep/lightstep-tracer-go/pull/34/files/aeaf1c009f2407c334d65da068706dabf1b3961e#r77910065>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADdiiUxL7SD8IS8ouJanPjHTLoOnKiJsks5qnzBygaJpZM4J3YRu>
> .
>